graviti/schema/builtin.py

```diff
@@ -306,6 +306,25 @@ def to_pyarrow(self) -> pa.DataType:
         """
         return pa.struct(self.fields.to_pyarrow())  # pylint: disable=no-member
 
+    def get_keys(self) -> List[Tuple[str, ...]]:
+        """Get the keys of each field to create lazy lists.
+
+        Returns:
+            The keys to create lazy lists.
```

Comment:
paging list?
Also I think we can use the keys to do other things not only create paging list.
Should we mention paging list in the doc?